### PR TITLE
dx-111 emphasized that callback is mandatory

### PR DESCRIPTION
**Pull request by marinafrydenb**

### CODE REVIEW

File: _includes/callback-reference.md [hunk 1]
```diff
@@ -7,11 +7,19 @@ When a change or update from the back-end system are made on a payment or
...
-  recommended. If a payer closes the browser window, a network error or
-  something else happens that prevents the payer from being redirect from
-  Swedbank Pay back to the merchant website, the callback is what ensures that
-  you receive information about what happened with the payment.
+Setting a `callbackUrl` in the HTTP `POST` API is **mandatory**. Below there
+are three example scenarios to why this is important:
+
+1. If the payer closes the payment window, the merchant will never know what
+    happened to the payment if `callbackUrl` is not implemented.
+2. If the payer stops up in a payment app such as Vipps or Swish, the payer
```
Comment (payex-dev) [inline _includes/callback-reference.md:15]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5268726)

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 4
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5268726&bid=17259579)

File: _includes/callback-reference.md [hunk 2]
```diff
@@ -7,11 +7,19 @@ When a change or update from the back-end system are made on a payment or
...
 
-* Setting a `callbackUrl` in the HTTP `POST` API is optional, but highly
-  recommended. If a payer closes the browser window, a network error or
-  something else happens that prevents the payer from being redirect from
-  Swedbank Pay back to the merchant website, the callback is what ensures that
-  you receive information about what happened with the payment.
+Setting a `callbackUrl` in the HTTP `POST` API is **mandatory**. Below there
+are three example scenarios to why this is important:
+
+1. If the payer closes the payment window, the merchant will never know what
```
Comment (payex-dev) [inline _includes/callback-reference.md:13]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5268726)

File: _includes/callback-reference.md [hunk 3]
```diff
@@ -7,11 +7,19 @@ When a change or update from the back-end system are made on a payment or
...
+    happened to the payment if `callbackUrl` is not implemented.
+2. If the payer stops up in a payment app such as Vipps or Swish, the payer
+    will never come back to the merchant. This means that the merchant won't know what
+    happened to the payment unless `callbackUrl` is implemented.
+3. If a payer closes the browser window, a network error or
```
Comment (payex-dev) [inline _includes/callback-reference.md:18]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5268726)

File: _includes/callback-reference.md [hunk 4]
```diff
@@ -7,11 +7,19 @@ When a change or update from the back-end system are made on a payment or
...
-  Swedbank Pay back to the merchant website, the callback is what ensures that
-  you receive information about what happened with the payment.
+Setting a `callbackUrl` in the HTTP `POST` API is **mandatory**. Below there
+are three example scenarios to why this is important:
+
+1. If the payer closes the payment window, the merchant will never know what
+    happened to the payment if `callbackUrl` is not implemented.
+2. If the payer stops up in a payment app such as Vipps or Swish, the payer
+    will never come back to the merchant. This means that the merchant won't know what
+    happened to the payment unless `callbackUrl` is implemented.
```
Comment (payex-dev) [inline _includes/callback-reference.md:17]: Codacy Issue found: [[list-item-spacing] Missing new line after list item](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5268726)

File: _includes/callback-reference.md [hunk 5]
```diff
@@ -7,11 +7,19 @@ When a change or update from the back-end system are made on a payment or
 transaction, Swedbank Pay will perform a callback to inform the payee (merchant)
 about this update.
 
-* Setting a `callbackUrl` in the HTTP `POST` API is optional, but highly
-  recommended. If a payer closes the browser window, a network error or
-  something else happens that prevents the payer from being redirect from
-  Swedbank Pay back to the merchant website, the callback is what ensures that
-  you receive information about what happened with the payment.
+Setting a `callbackUrl` in the HTTP `POST` API is **mandatory**. Below there
+are three example scenarios to why this is important:
```
Comment (asbjornu) [inline _includes/callback-reference.md:11]:
```suggestion
Providing a `callbackUrl` in `POST` requests is **mandatory**. Below we provide
three example scenarios of why this is important:
```

File: _includes/callback-reference.md [hunk 6]
```diff
@@ -7,11 +7,19 @@ When a change or update from the back-end system are made on a payment or
...
+
+1. If the payer closes the payment window, the merchant will never know what
+    happened to the payment if `callbackUrl` is not implemented.
+2. If the payer stops up in a payment app such as Vipps or Swish, the payer
+    will never come back to the merchant. This means that the merchant won't know what
+    happened to the payment unless `callbackUrl` is implemented.
+3. If a payer closes the browser window, a network error or
+    something else can happen that prevents the payer from being redirected from
+    Swedbank Pay back to the merchant website. The `callbackUrl` is what ensures that
+    you receive the information about what happened with the payment.
```
Comment (asbjornu) [inline _includes/callback-reference.md:21]:
```suggestion
3. If a payer experiences a network error or something else happens that
   prevents the payer from being redirected from Swedbank Pay back to the
   merchant website, the `callbackUrl` is what ensures that you receive the
   information about what happened with the payment.
```

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 4
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5268726&bid=17259579)

File: _includes/callback-reference.md [hunk 7]
```diff
@@ -7,11 +7,19 @@ When a change or update from the back-end system are made on a payment or
...
-  recommended. If a payer closes the browser window, a network error or
-  something else happens that prevents the payer from being redirect from
-  Swedbank Pay back to the merchant website, the callback is what ensures that
-  you receive information about what happened with the payment.
+Providing a `callbackUrl` in `POST` requests is **mandatory**. Below we provide
+three example scenarios of why this is important:
+
+1. If the payer closes the payment window, the merchant will never know what
+    happened to the payment if `callbackUrl` is not implemented.
+2. If the payer stops up in a payment app such as Vipps or Swish, the payer
```
Comment (payex-dev) [inline _includes/callback-reference.md:15]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5268726)

File: _includes/callback-reference.md [hunk 8]
```diff
@@ -7,11 +7,19 @@ When a change or update from the back-end system are made on a payment or
...
 
-* Setting a `callbackUrl` in the HTTP `POST` API is optional, but highly
-  recommended. If a payer closes the browser window, a network error or
-  something else happens that prevents the payer from being redirect from
-  Swedbank Pay back to the merchant website, the callback is what ensures that
-  you receive information about what happened with the payment.
+Providing a `callbackUrl` in `POST` requests is **mandatory**. Below we provide
+three example scenarios of why this is important:
+
+1. If the payer closes the payment window, the merchant will never know what
```
Comment (payex-dev) [inline _includes/callback-reference.md:13]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5268726)

File: _includes/callback-reference.md [hunk 9]
```diff
@@ -7,11 +7,19 @@ When a change or update from the back-end system are made on a payment or
...
+    happened to the payment if `callbackUrl` is not implemented.
+2. If the payer stops up in a payment app such as Vipps or Swish, the payer
+    will never come back to the merchant. This means that the merchant won't know what
+    happened to the payment unless `callbackUrl` is implemented.
```
Comment (payex-dev) [inline _includes/callback-reference.md:17]: Codacy Issue found: [[list-item-spacing] Missing new line after list item](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5268726)

File: _includes/callback-reference.md [hunk 10]
```diff
@@ -7,11 +7,19 @@ When a change or update from the back-end system are made on a payment or
...
+3. If a payer experiences a network error or something else happens that
```
Comment (payex-dev) [inline _includes/callback-reference.md:18]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5268726)

**Pull request merged**